tensorflow_probability/python/bijectors/masked_autoregressive_test.py

```diff
@@ -729,6 +729,60 @@ def test_doc_string(self):
     self.assertAllEqual(
         (3,), distribution.log_prob(np.ones((3, 2), dtype=np.float32)).shape)
 
+  def test_doc_string_2(self):
+    np.random.seed(123)
```

Comment:
No need to set the seed, it's set somewhere deep in TensorFlow's base testcase.

Reply:
Ah yes, I wondered about this but didn't want to assume. Thanks